examples/open-records/open_records.bal

```diff
@@ -46,11 +46,11 @@ public function main() {
     s2["credits"] = 120.5;
     io:println(s2);
 
-    // You can assign a `PartTimeStudent` type value to a `Student`.
+    // You can assign a `PartTimeStudent`-typed value to a `Student`-typed variable.
     Student s3 = s2;
     io:println(s3);
 
-    // You can assign a `Person` type value to a `map`.
+    // You can assign a `Student`-typed value to a `map<anydata>`-typed variable.
```

Comment:
Is `Student`-typed value correct here though? The value is actually `Student`, right?

I usually use

```suggestion
    // A variable of type `Student` can be used where a `map<anydata>` value is expected.
```

Reply:
Changed the other comment as well for consistency.